app/services/metrics/defect_escape_rate/per_project.rb

```diff
@@ -27,9 +27,14 @@ def query(interval)
         ]]
       end
 
+      # :reek:DuplicateMethodCall
       def bugs_by_environment(bug_issues)
         bug_issues.each_with_object({}) do |bug, result|
```

Comment:
Maybe we can set a default for these keys? Something like...

```suggestion
        bug_issues.each_with_object(Hash.new(1)) do |bug, result|
```

Reply:
Agree with Hora, but actually, I think it should start in 0 by default and always add 1

```ruby
bug_issues.each_with_object(Hash.new(0)) do |bug, result|
  result[bug.environment] += 1
end
```

Reply:
Nice!